src/generate/temporal.ts

```
@@ -0,0 +1,1093 @@
+  let day: number | undefined;
+  let hour: number | undefined;
+  let minute: number | undefined;
+  let second: number | undefined;
+  let millisecond: number | undefined;
+  let quarter: number | undefined;
+  let meridiem: string | undefined;
+  let parsedWeek: number | undefined;
+  let parsedWeekYear: number | undefined;
+  let parsedWeekType: 'locale' | 'iso' | undefined;
+  let parsedWeekYearType: 'locale' | 'iso' | undefined;
+  let parsedWeekDays: Array<{ token: 'dddd' | 'ddd' | 'dd'; value: string }> = [];
```

ESLint

[error] 'parsedWeekDays' is never reassigned. Use 'const' instead.

(prefer-const)